boards/matek/h743/init/rc.board_defaults

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+# board specific defaults
+#------------------------------------------------------------------------------
+
+param set-default BAT2_V_DIV 11
+param set-default BAT2_A_PER_V 40
+
+param set-default BAT1_V_DIV 11.0
+param set-default BAT1_A_PER_V 40.0
+
+# Disable safety switch by default
+param set-default CBRK_IO_SAFETY 22027
```

Comment:
It's off by default now.

Reply:
```suggestion
```

Reply:
The board does not come with safety switches, same as non native boards like the `omnibus f4sd`

Reply:
oh like the parameter is off by default okay i'll take it out

Reply:
Right, it's legacy cruft we should stop propagating everywhere.

Reply:
fixed with [0c9f6ac](https://github.com/matthewoots/PX4-Autopilot/commit/0c9f6acc4828159cec01f01d4a0ebda59d29fae9)